hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/AWSStatus500Exception.java

```diff
@@ -29,7 +29,7 @@
  *    <li>Nothing else got through either.</li>
  * </ol>
  */
-public class AWSStatus500Exception extends AWSServiceIOException {
+public class jAWSStatus500Exception extends AWSServiceIOException {
```

Comment:
I think this is a typo in Intellij.

Reply:
And I think Yetus failed beacuse of this only.

Reply:
aah